pkg/utils/accelerators/tpu.go

```diff
@@ -211,6 +216,10 @@ func AddTPUVariables(pod *corev1.Pod, size int) error {
 			Name:  TpuWorkerId,
 			Value: fmt.Sprint(tpuWorkerId),
 		},
+		corev1.EnvVar{
+			Name:  TpuName,
+			Value: fmt.Sprint(pod.Name),
```

Comment:
I think this value is used by Ray to know which TPU host belongs to which slice? So if it's a multi-host slice, the value that be the same.  So one replica has one name, let's use leader pod's name here